### PR TITLE
create translation files (german)

### PR DESCRIPTION
**Pull request by rschnapka**
intruduce german translation

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: barcode_link/i18n/de.po [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: de\n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: barcode_link
+#: model:ir.module.module,description:barcode_link.module_meta_information
+msgid " This module is the Barcode link module"
+msgstr "Dieses Modul stellt stellt Barcode-Verknüpfungen her"
```
Comment (yvaucher) [inline barcode_link/i18n/de.po]: I think there is a typo here "stellt" is written twice for herstellen.

**Comment by yvaucher** [comment]
@rschnapka Thanks again for you contribution. I think there is a little typo. Otherwise :+1:

**Comment by StefanRijnhart** [comment]
Thanks! I see you followed up @yvaucher's comment about the typo, but you did not leave a comment about it, so @yvaucher was probably not aware of that change. Seeing that the typo was fixed, I'll merge.

**Comment by yvaucher** [comment]
Thanks @StefanRijnhart , indeed I was not noticed

**Pull request merged**